v2/yarpcrouter/router.go

```diff
@@ -54,23 +54,26 @@ type MapRouter struct {
 }
 
 // NewMapRouter builds a new MapRouter that uses the given name as the
-// default service name.
-func NewMapRouter(defaultService string) MapRouter {
-	return MapRouter{
+// default service name and registers the given procedures.
+//
+// If a procedure does not specify its service name, the procedure will
```

Comment:
Since we're here, let's make this comment more digestible

```
 Procedures that do not specify a service name will inherit the default service
 name of the router. Multiple procedures with the same name and service name can
 only exist if they handle different encodings. If a procedure does not specify
 an encoding, it can only support one handler; the router will select that
 handler regardless of encoding.
```